third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -51,29 +53,31 @@ pub fn run_move_compiler(
     error_writer: &mut impl WriteColor,
     options: Options,
 ) -> anyhow::Result<(GlobalEnv, Vec<AnnotatedCompiledUnit>)> {
+    logging::setup_logging();
+    info!("Move Compiler v2");
     // Run context check.
     let mut env = run_checker(options.clone())?;
     check_errors(&env, error_writer, "checking errors")?;
-
     function_checker::check_for_function_typed_parameters(&mut env);
     function_checker::check_access_and_use(&mut env);
     check_errors(&env, error_writer, "checking errors")?;
 
     if options.debug {
```

Comment:
I am assuming we want to eventually get rid of these conditionals, as `debug!` is already conditiontal.

Reply:
Yes, we should have a discussion what we do next here, I'll also open a slack thread for it shortly.